docs/usage/self-hosted-configuration.md

```diff
@@ -997,7 +1009,7 @@ Defines how the report is exposed:
 - `<unset>` If unset, no report will be provided, though the debug logs will still have partial information of the report
 - `logging` The report will be printed as part of the log messages on `INFO` level
 - `file` The report will be written to a path provided by [`reportPath`](#reportpath)
-- `s3` The report is pushed to an S3 bucket defined by [`reportPath`](#reportpath). This option reuses [`RENOVATE_X_S3_ENDPOINT`](./self-hosted-experimental.md#renovatexs3endpoint) and [`RENOVATE_X_S3_PATH_STYLE`](./self-hosted-experimental.md#renovatexs3pathstyle)
+- `s3` The report is pushed to an S3 bucket defined by [`reportPath`](#reportpath). This option reuses [`RENOVATE_X_S3_ENDPOINT`](./self-hosted-experimental.md#renovatexs3endpoint) and [`s3PathStyle`](./self-hosted-experimental-flags.md#s3pathstyle)
```

Comment:
this will be highly confusing. some parts need to be env and one a flag.

this should be avoided.

Reply:
@nabeelsaabna it would be good to make these proper config options for v38

Reply:
from the ones changed in this PR i only see one that we use `RENOVATE_X_EXEC_GPID_HANDLE`

Reply:
actually two:
```
RENOVATE_X_DOCKER_HUB_TAGS -> dockerHubTags
RENOVATE_X_EXEC_GPID_HANDLE -> execGpidHandle
```
its good to have a summary at the end in the PR body